generic_hw_control_loop:
  loop_hz: 300
  cycle_time_error_threshold: 0.01


hardware_interface:
  joints:
    - joint1
    - joint2
  sim_control_mode: 0


joint_state_controller:
  type:
    joint_state_controller/JointStateController
    publish_rate: 50

position_trajectory_controller:
  type: position_controllers/JointTrajectoryController
  joints:
    - joint1
    - joint2
  constraints:
    goal_time: 5.0
    joint1:
      trajectory: 0.60
      goal: 0.15
    joint2:
      trajectory: 0.60
      goal: 0.15
  #  gains:
  #    joint1: {p: 2.0, i: 0.0, d: 0.01, i_clamp: 1}
  #    joint2: {p: 2.0, i: 0.0, d: 0.01, i_clamp: 1}